mealie/routes/admin/admin_invitations.py

```diff
@@ -0,0 +1,25 @@
+from fastapi import APIRouter
+
+from mealie.routes._base import BaseAdminController, controller
+from mealie.schema.household.invite_token import (
+    ReadInviteToken,
+)
+from mealie.schema.response.pagination import PaginationQuery
+
+router = APIRouter(prefix="/invitations")
+
+
+@controller(router)
+class AdminInvitationsController(BaseAdminController):
+    @router.get("", response_model=list[ReadInviteToken])
+    def get_all(self):
+        """Get all invite tokens"""
+        self.repos.group_id = None
+        self.repos.household_id = None
+        return self.repos.group_invite_tokens.page_all(PaginationQuery(page=1, per_page=-1)).items
+
+    @router.get("/{group_slug}", response_model=list[ReadInviteToken])
+    def get_one(self, group_slug: str):
+        """Get all invite tokens for a specific group"""
+        self.repos.group_id = self.repos.groups.get_by_slug_or_id(group_slug).id
+        return self.repos.group_invite_tokens.page_all(PaginationQuery(page=1, per_page=-1)).items
```

Comment:
We can get rid of this route; `get_one` implies you're getting a single invitation (rather than all invitations for a particular group). At the admin-level I don't think this is necessary. If we do need this functionality I'd rather build it into the `get_all` route using pagination, rather than having a separate route for it.

Reply:
J'ai supprimé cette route et ajouté `PaginationQuery` dans `get_all`